cypress/integration/landing.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { examples } from '@components/SearchExamples/examples';
+import { themes } from '@components/NavBar/models';
+
+const baseUrl = Cypress.config().baseUrl;
+
+const themeOptions = Object.values(themes);
+
+describe('Landing Page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('form').find('input[type="text"]', { timeout: 10000 }).first().as('input').should('be.enabled');
+  });
```

Comment:
This is okay, although I would say that we may wanna not do this in a unit test style, since it increases the run time significantly.
Probably we could improve on the `results.spec` file in this way too.
If we can structure them so that we only do a `cy.visit` a minimum amount of times that would be ideal.

Reply:
I have made changes to make it more like integration tests.